### ISSUE TITLE
Run on Azure Function App

### ISSUE THREAD

**Issue by BorisShulmanTabit** [weizhonzhen/FastHtmlToPdf#2]
Running of the html to pdf converter on Azure Function  App and .Net Core on Win gave different result. In 1st case all letters are presented as black squares. After run function  'Convert' the size of byte array are different. When running on Azure Function App the size is bigger

**Comment by weizhonzhen** [comment]
> Running of the html to pdf converter on Azure Function App and .Net Core on Win gave different result. In 1st case all letters are presented as black squares. After run function 'Convert' the size of byte array are different. When running on Azure Function App the size is bigger

1、&lt;head&gt;&lt;meta charset="utf-8"&gt;&lt;/head&gt;
2、Encoding.UTF8

**Comment by BorisShulmanTabit** [comment]
@weizhonzhen , sorry, it is my fault to write it as an issue here, because Your code works perfect in .Net Core. I found the solution for an issue that I wrote and it is not a code issue. 
resolution is here: [https://stackoverflow.com/questions/53188629/how-to-show-text-in-pdf-generated-by-azure-function](https://stackoverflow.com/questions/53188629/how-to-show-text-in-pdf-generated-by-azure-function) . 
Once again sorry, Your code is perfect. It is significantly easier to work with than with Dink.
Thank You
